composer/algorithms/seq_length_warmup/seq_length_warmup.py

```diff
@@ -64,25 +67,31 @@ def set_batch_sequence_length(batch: Dict[str, torch.Tensor], curr_seq_len: int,
     """
 
     assert isinstance(batch, Mapping)
+    assert 'attention_mask' in batch
```

Comment:
Can we raise a ValueError if the `attention_mask` isn't in the batch?

Reply:
Done